kodilla-patterns/src/test/java/com/kodilla/patterns/builder/bigmac/BigmacTestSuite.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BigmacTestSuite {
+
+    @Test
+    void testNweBigmac() {
+        Bigmac bigmac = new Bigmac.BigmacBuldier()
+                .bun("no sesame")
+                .burgers(2)
+                .sauce("1000 Island dressing")
+                .ingredient("cucumber")
+                .ingredient("onion")
+                .build();
+
+        int howManyIngredient = Bigmac.getIngredients().size();
+
+        assertEquals(2, howManyIngredient);
```

Comment:
assertEquals("no sesame", bigmac.getBun());
assertEquals(2, bigmac.getBurgers());
assertEquals(List.of("cucumber", "onion"), bigmac.getIngredients());
assertEquals("1000 Island dressing", bigmac.getSauce());